examples/workflows/upstream/ci.py

```diff
@@ -0,0 +1,140 @@
+from hera.workflows import Container, Step, Steps, Workflow
+from hera.workflows.models import (
+    Arguments,
+    Artifact,
+    GitArtifact,
+    Inputs,
+    Item,
+    ObjectMeta,
+    Parameter,
+    PersistentVolumeClaim,
+    PersistentVolumeClaimSpec,
+    Quantity,
+    VolumeMount,
+    VolumeResourceRequirements,
+)
+
+with Workflow(
+    arguments=Arguments(
+        parameters=[
+            Parameter(
+                name="revision",
+                value="cfe12d6",
```

Comment:
Would be nice to avoid generating trailing commas so the formatter can decide whether it wants to collapse onto a single line.